internal/docker/testcafe.go

```diff
@@ -0,0 +1,63 @@
+package docker
+
+import (
+	"context"
+
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/cli/command"
+	"github.com/saucelabs/saucectl/internal/framework"
+	"github.com/saucelabs/saucectl/internal/testcafe"
+)
+
+// TestcafeRunner represents the docker implementation of a test runner.
+type TestcafeRunner struct {
+	ContainerRunner
+	Project testcafe.Project
+}
+
+// NewTestcafe creates a new TestcafeRunner instance.
+func NewTestcafe(c testcafe.Project, cli *command.SauceCtlCli, imageLoc framework.ImageLocator) (*TestcafeRunner, error) {
+	r := TestcafeRunner{
+		Project: c,
+		ContainerRunner: ContainerRunner{
+			Ctx:             context.Background(),
+			Cli:             cli,
+			containerID:     "",
```

Comment:
Just leave out those values if you aren't going to set them :)

Reply:
updating.